microsoft-edge/webview2/how-to/print.md

```diff
@@ -307,7 +306,7 @@ async void PrintToPrinter()
 
     if (printStatus == CoreWebView2PrintStatus.Succeeded)
     {
-      MessageBox.Show(this, "Printing " + title + " document to printer is succeeded", "Print to printer");
+      MessageBox.Show(this, "Printing " + title + " document to printer succeeded", "Print to printer");
```

Comment:
@monica-ch -- innocuous change of a display string in code.  Alternatives:

document to printer succeeded.
document to printer was successful.